includes/themes.php

```diff
@@ -964,6 +966,26 @@ public function install_theme( $file ) {
 	public function sync_settings() {
 		_deprecated_function( __METHOD__, '5.0' );
 	}
+
+	/**
+	 * Check current theme requires plugin version and show admin notice if it doesn't meet the requirement  .
+	 *
+	 * @returns {void}
+	 */
+	private function check_theme_requiers_version() {
+		$theme_requires = $this->get_active_theme_data( 'requires' );
+		if ( defined( 'WPBDP_VERSION' ) && $theme_requires && version_compare( WPBDP_VERSION, $theme_requires, '<' ) ) {
+			add_action( 'admin_notices', array( $this,'update_bd_admin_notice' ) );
```

Comment:
Does this work if we add is_admin() to the checks and move the new wpbdp_admin_message line here instead of using a hook? It would be nice if we didn't have to add a new public function if we don't have to since private functions are easier to maintain and remove.

Reply:
We can use leave the hook then. Thanks for checking!